web/src/main/java/it/pagopa/selfcare/dashboard/web/controller/InstitutionV2Controller.java

```diff
@@ -76,4 +78,22 @@ public List<InstitutionBaseResource> getInstitutions(Authentication authenticati
 
         return result;
     }
+
+    @GetMapping("/{institutionId}")
+    @ResponseStatus(HttpStatus.OK)
+    @ApiOperation(value = "", notes = "${swagger.dashboard.institutions.api.getInstitution}")
```

Comment:
you could fill values with ${swagger.dashboard.institutions.api.getInstitution} and add nickname value to set operationId value, for example "v2GetInstitution